code/modules/antagonists/heretic/items/heretic_grenade.dm

```diff
@@ -0,0 +1,111 @@
+/*!
+ * Contains Heretic grenades
+ * They spread rust and obliterate borgs/mechs
+ */
+
+/obj/item/grenade/chem_grenade/rust_sower
+	name = "\improper Rust sower"
+	desc = "A nifty little thing that explodes into rust. Causes borgs and mechs to get utterly obliterated"
+	possible_fuse_time = list("5")
+	stage = GRENADE_READY
+	base_icon_state = "rustgrenade"
+	grenade_arm_sound = 'sound/items/weapons/rust_sower_armbomb.ogg'
+	grenade_sound_vary = FALSE
+
+/obj/item/grenade/chem_grenade/rust_sower/update_icon_state()
+	. = ..()
+	if(active)
+		icon_state = "[base_icon_state]_active"
+	else
+		icon_state = base_icon_state
+
+/obj/item/grenade/chem_grenade/rust_sower/Initialize(mapload)
+	. = ..()
+	var/obj/item/reagent_containers/cup/beaker/large/beaker_one = new(src)
+	var/obj/item/reagent_containers/cup/beaker/large/beaker_two = new(src)
+
+	beaker_one.reagents.add_reagent(/datum/reagent/heretic_rust, 50)
+	beaker_one.reagents.add_reagent(/datum/reagent/potassium, 50)
+	beaker_two.reagents.add_reagent(/datum/reagent/phosphorus, 50)
+	beaker_two.reagents.add_reagent(/datum/reagent/consumable/sugar, 50)
+
+	beakers += beaker_one
+	beakers += beaker_two
+
+/obj/item/grenade/chem_grenade/rust_sower/detonate(mob/living/lanced_by)
+	. = ..()
+	playsound(src, 'sound/items/weapons/rust_sower_explode.ogg', 70, FALSE)
+	qdel(src)
+
+/obj/item/grenade/chem_grenade/rust_sower/screwdriver_act(mob/living/user, obj/item/tool)
+	return FALSE
+
+/obj/item/grenade/chem_grenade/rust_sower/wrench_act(mob/living/user, obj/item/tool)
+	return FALSE
+
+/obj/item/grenade/chem_grenade/rust_sower/multitool_act(mob/living/user, obj/item/tool)
+	return FALSE
+
+/datum/reagent/heretic_rust
+	name = "Eldritch Rust"
+	description = "A slurry of some off-putting liquid. Just looking at it makes you feel sick."
+	color = COLOR_CARGO_BROWN // Rust color
+	taste_description = "scorching agony"
```

Comment:
```suggestion
	description = "A slurry of viscous, chunky brown liquid."
	color = COLOR_CARGO_BROWN // Rust color
	taste_description = "rotten copper"
```
The 'description' only shows up in chem analyzers and chem masters, probably shouldn't have subjective disgust